src/test/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClientTestCI.java

```diff
@@ -32,7 +32,7 @@
     private static final int BANYANDB_PORT = 17912;
     @Rule
     public GenericContainer<?> banyanDB = new GenericContainer<>(
-            DockerImageName.parse(parseImageName()))
+            DockerImageName.parse("ghcr.io/apache/skywalking-banyandb:d061ab4abe0232c868f60cd3f311877b5a3703ac"))
```

Comment:
This should be separated as static fields, including image domain, image name, version